docs/src/inputs/select/examples/exampleClearableSelect.jsx

```diff
@@ -0,0 +1,29 @@
+import { isEmpty } from 'lodash';
+import React, { useState } from 'react';
+import {
+    Select,
+} from 'react-cm-ui';
+import { options } from '../constants';
+
+function ExampleClearableSelect() {
+    const [selectedValue, setOption] = useState({});
+
+    function onSelect(selectedOption) {
+        setOption(selectedOption);
+    }
+
+    return (
+        <div>
```

Comment:
Are the surrounding `<div>`s in all of these examples intentional?

Reply:
Nah.

Reply:
fixed.

Reply:
Resolving.